src/app/app-routing.module.ts

```diff
@@ -1,13 +1,21 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { MainListComponent } from './components/main-list/main-list.component';
 import { AdminFormCreateEventComponent } from './components/admin-form-create-event/admin-form-create-event.component';
 import { AdminFormManageEventComponent } from './components/admin-form-manage-event/admin-form-manage-event.component';
 
 
 const routes: Routes = [
+ 
+  {
+    path: 'mainList', component: MainListComponent
+  },
   { path: 'new/advertisingconfig', component: AdminFormCreateEventComponent },
   { path: 'manage/advertisingconfig', component: AdminFormManageEventComponent}
-];
+
+
+
+
```

Comment:
looks like you are missing a closing array here

Reply:
Thank you @robertlb4 ! Changes made here and remove faker file in the package.json file.